test/syscalls/linux/kill.cc

```diff
@@ -102,6 +102,57 @@ TEST(KillTest, CanKillAllPIDs) {
   EXPECT_EQ(0, WEXITSTATUS(status));
 }
 
+// reaper process of pid namespace will ignore blockable
+// signal with SIG_DFL action, such as SIGTERM
+TEST(KillTest, CannotTermReaper) {
```

Comment:
Can we test this in a pid namespace?  Trying to kill pid 1 seems dangerous.

Reply:
I have re-write the case, run the case in new pid namespace.